plugins/woocommerce/src/Blocks/Patterns/PTKClient.php

```diff
@@ -14,6 +14,68 @@ class PTKClient {
 	 */
 	const PATTERNS_TOOLKIT_URL = 'https://public-api.wordpress.com/rest/v1/ptk/patterns/';
 
+	/**
+	 * The schema for the patterns toolkit.
+	 *
+	 * @var array
+	 */
+	private $schema;
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		$this->schema = [
+			'type'     => 'array',
+			'items'    => [
+				'type'       => 'object',
+				'properties' => [
+					'ID'         => [
+						'type' => 'integer',
+					],
+					'site_id'    => [
+						'type' => 'integer',
+					],
+					'title'      => [
+						'type' => 'string',
+					],
+					'name'       => [
+						'type' => 'string',
+					],
+					'html'       => [
+						'type' => 'string',
+					],
+					'categories' => [
+						'type'                 => 'object',
+						'additionalProperties' => [
```

Comment:
https://developer.wordpress.org/rest-api/extending-the-rest-api/schema/#:~:text=%2C%0A)%3B-,additionalProperties,-Perhaps%20unintuitively%2C%20by